.github/workflows/build.yml

```diff
@@ -51,6 +51,9 @@ jobs:
         mkdir .cargo
         echo '[target.x86_64-pc-windows-msvc]' >> .cargo/config
         echo 'rustflags = ["-Ctarget-feature=+crt-static"]' >> .cargo/config
+    - name: Run the application without starting the shell
+      run: |
+        sed -i '1 i\#![windows_subsystem = \"windows\"]' examples/todos/src/main.rs
```

Comment:
I think we can just add this line to the `main` file directly.

Reply:
I think that if we add that attribute to the `main` file directly we will prevent developers to start the shell. Perhaps we could do that only for release builds

Reply:
Good point! Let's merge then.

Reply:
Thank you for merging my PR! :)